app/transforms/cordova-basic-array.js

```diff
@@ -8,7 +9,7 @@ import Ember from 'ember';
 export default DS.Transform.extend({
   deserialize: function (serialized) {
     return Ember.ArrayProxy.create({
-      content: serialized ? Ember.EnumerableUtils.map(serialized, function (item) {
+      content: serialized ? map(serialized, function (item) {
```

Comment:
This could be done without including lodash (including lodash adds another lib and so increase size of the application at the end). I guess something like this would do it:
```js
function(){
  var res = [];
  for (var i = 0; i < serialized.length; i++) res.push(Ember.ObjectProxy.create({content: serialized[i] || {}}));
  return res;
}(serialized)
```